maze_transformer/training/training.py

```diff
@@ -35,175 +33,85 @@ class TRAIN_SAVE_FILES:
     model_final: str = "model.final.pt"
 
 
-def setup_logger(output_path: Path, config: ConfigHolder) -> Logger:
-    logger: Logger = Logger(
-        log_path=Path(output_path / "log.jsonl").as_posix(),
-        console_print_threshold=30,
-        streams=(
-            LoggingStream(
-                name="log_config", aliases=("cfg", "config"), default_level=40
-            ),
-            LoggingStream(name="train", aliases=("log_train",), default_level=50),
-            LoggingStream(
-                name="mem_usage",
-                aliases=("traced_memory", "mem"),
-                default_level=40,
-                default_contents={
-                    "traced_memory": (
-                        lambda: dict(
-                            zip(("current", "peak"), tracemalloc.get_traced_memory())
-                        )
-                    )
-                },
-            ),
-        ),
-    )
-
-    logger.log("loaded data config, initialized logger")
-    logger.log_config(json_serialize(config))
-    logger.log_config(
-        dict(
-            logger_cfg={
-                "output_dir": output_path,
-                "data_cfg.name": config.dataset_cfg.name,
-                "train_cfg.name": config.train_cfg.name,
-                "model_cfg.device": config.model_cfg.name,
-            },
-            lvl=0,
-        )
-    )
-
-    return logger
-
-
 def get_dataloader(
-    dataset: MazeDataset, cfg: ConfigHolder, logger: Logger
+    dataset: MazeDataset, cfg: ConfigHolder, logger: WandbLogger
 ) -> DataLoader:
-    logger.log_elapsed_last()
-    logger.mem_usage()
-
     length_stats: StatCounter = StatCounter(dataset.get_all_lengths())
-    logger.log({"dataset_seq_len_stats": length_stats.summary()})
-    logger.log({"dataset_seq_len_stats": length_stats.serialize()}, lvl=50)
+    logger.summary({"dataset_seq_len_stats_summary": length_stats.summary()})
```

Comment:
<img width="678" alt="image" src="https://user-images.githubusercontent.com/51809771/227679803-32970d52-3edf-4110-ba15-c9c76fcefc78.png">
Looks like the serialized version below already includes the summary?

Reply:
Yes it does. I think for now I might just log the summary to the console so it doesn't clutter up the wandb summary